[2025-10-06T14:43:19.702+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-06T14:43:19.719+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.run_pyspark_task manual__2025-10-06T14:43:18.801775+00:00 [queued]>
[2025-10-06T14:43:19.726+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.run_pyspark_task manual__2025-10-06T14:43:18.801775+00:00 [queued]>
[2025-10-06T14:43:19.727+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-06T14:43:19.739+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_pyspark_task> on 2025-10-06 14:43:18.801775+00:00
[2025-10-06T14:43:19.746+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=8745) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-06T14:43:19.747+0000] {standard_task_runner.py:63} INFO - Started process 8747 to run task
[2025-10-06T14:43:19.748+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'main', 'run_pyspark_task', 'manual__2025-10-06T14:43:18.801775+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpx08k_gvt']
[2025-10-06T14:43:19.749+0000] {standard_task_runner.py:91} INFO - Job 155: Subtask run_pyspark_task
[2025-10-06T14:43:19.784+0000] {task_command.py:426} INFO - Running <TaskInstance: main.run_pyspark_task manual__2025-10-06T14:43:18.801775+00:00 [running]> on host 5adecdfd8c83
[2025-10-06T14:43:19.847+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='run_pyspark_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T14:43:18.801775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-06T14:43:18.801775+00:00'
[2025-10-06T14:43:19.848+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-06T14:43:32.140+0000] {logging_mixin.py:188} INFO - +--------+------------------+------------------+----------------+--------+----------+-------------------+--------------------+--------------------+--------------------+-------------------+--------------------+
|house_id|          latitude|         longitude|maintenance_year|  square|population|             region|       locality_name|             address|        full_address|communal_service_id|         description|
+--------+------------------+------------------+----------------+--------+----------+-------------------+--------------------+--------------------+--------------------+-------------------+--------------------+
|       1|         44.707617|         43.006476|            1974|2 661.10|        89|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                1.0|Жилой дом в , по ...|
|       2|44.706720000000004|         43.005281|            1989|3 111.10|       115|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                1.0|Жилой дом в , по ...|
|       3|         44.708302|43.006665000000005|            1977|2 896.04|        89|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                1.0|Жилой дом в , по ...|
|       5|44.711946999999995|          42.97358|            1978|1 061.92|        26|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                1.0|Жилой дом в , по ...|
|       6|         44.707751|         43.005515|            1978|2 688.70|       105|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                2.0|Жилой дом в , по ...|
|       7|44.711946999999995|          42.97358|            1973|  854.90|        40|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                3.0|Жилой дом в , по ...|
|       8|44.723622999999996|         42.996864|            1978|  924.40|        31|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                4.0|Жилой дом в , по ...|
|       9|44.721765999999995|          43.02013|            1974|1 011.30|        38|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                5.0|Жилой дом в , по ...|
|      10|         44.710557|         43.000278|            1963|  621.21|        20|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                6.0|Жилой дом в , по ...|
|      11|44.724903999999995|         43.003359|            1972|  681.60|        26|Ставропольский край|село Александровское|с. Александровско...|Ставропольский кр...|                7.0|Жилой дом в , по ...|
+--------+------------------+------------------+----------------+--------+----------+-------------------+--------------------+--------------------+--------------------+-------------------+--------------------+
only showing top 10 rows
[2025-10-06T14:43:40.384+0000] {logging_mixin.py:188} INFO - Количество строк загруженных из файла 507914
[2025-10-06T14:43:40.386+0000] {logging_mixin.py:188} INFO - root
 |-- house_id: integer (nullable = true)
 |-- latitude: double (nullable = true)
 |-- longitude: double (nullable = true)
 |-- maintenance_year: string (nullable = true)
 |-- square: string (nullable = true)
 |-- population: string (nullable = true)
 |-- region: string (nullable = true)
 |-- locality_name: string (nullable = true)
 |-- address: string (nullable = true)
 |-- full_address: string (nullable = true)
 |-- communal_service_id: string (nullable = true)
 |-- description: string (nullable = true)
[2025-10-06T14:43:40.439+0000] {logging_mixin.py:188} INFO - root
 |-- house_id: integer (nullable = true)
 |-- latitude: float (nullable = true)
 |-- longitude: float (nullable = true)
 |-- maintenance_year: integer (nullable = true)
 |-- square: float (nullable = true)
 |-- population: integer (nullable = true)
 |-- region: string (nullable = true)
 |-- locality_name: string (nullable = true)
 |-- address: string (nullable = true)
 |-- full_address: string (nullable = true)
 |-- communal_service_id: integer (nullable = true)
 |-- description: string (nullable = true)
[2025-10-06T14:43:53.874+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-06T14:43:53.874+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main.py", line 95, in run_spark_job
    engine = create_engine(
             ^^^^^^^^^^^^^^
  File "<string>", line 2, in create_engine
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/create.py", line 518, in create_engine
    entrypoint = u._get_entrypoint()
                 ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/url.py", line 662, in _get_entrypoint
    cls = registry.load(name)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py", line 343, in load
    raise exc.NoSuchModuleError(
sqlalchemy.exc.NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:clickhouse
[2025-10-06T14:43:53.884+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=main, task_id=run_pyspark_task, run_id=manual__2025-10-06T14:43:18.801775+00:00, execution_date=20251006T144318, start_date=20251006T144319, end_date=20251006T144353
[2025-10-06T14:43:53.894+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 155 for task run_pyspark_task (Can't load plugin: sqlalchemy.dialects:clickhouse; 8747)
[2025-10-06T14:43:53.931+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-06T14:43:53.943+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-06T14:43:53.945+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
